src/lib.rs

```diff
@@ -1,3 +1,3 @@
 #![allow(non_snake_case, unused)]
 
-include!(concat!(env!("OUT_DIR"), "/glue.rs"));
+include!(concat!(env!("OUT_DIR"), "/glue.rs"));
```

Comment:
It looks like `cargo fmt` doesn't like that there's not a new line at the end of file? That's a weird one.

Reply:
The make integration-tests are failing due to the path of the wasm file for the test, the make command cd's into the integration-tests directory, so it can't find this file.

```
wasm_module_path = os.path.abspath("integration-tests/smartmodule_filter_on_a.wasm")
```

I can remove the directory from the path, it was only there as I was using `python setup.py test` for quick feedback, perhaps creating a command to build and provide examples could replace the `python setup.py test` command?

Reply:
Oh, interesting. I honestly don't recall why I did `cd integration-tests && ../venv/bin/python -m unittest` rather than `python setup.py test`. In #160, I made CI use `python setup.py test` as it seems more idiomatic.